### PR TITLE
perldelta.pod: Add section for 'use VERSION' restrictions

### PR DESCRIPTION
**Pull request by leonerd**

### CODE REVIEW

**Comment by jkeenan** [comment]
Is this p.r. dependent on any other recent p.r.'s?  Or can it be merged as soon as it's reviewed?

**Comment by rwp0** [comment]
> Is this p.r. dependent on any other recent p.r.'s?  Or can it be merged as soon as it's reviewed?


Seems to me those two are committed already:

- https://github.com/Perl/perl5/commit/bcaa4f978cf94bb1637c7bd63431b321a46328ff

- https://github.com/Perl/perl5/commit/5fdf6e90221900a85cd3091bd2755e41fc230d4d

**Comment by leonerd** [comment]
These are deltas for already-merged commits, yes. I PR'ed it only for the usual checking of spelling/grammar/wordings etc..

File: pod/perldelta.pod [hunk 1]
```diff
@@ -66,6 +66,12 @@ declaration when another is in scope, when either version is 5.39 or above.
 This is to avoid complications surrounding imported lexical functions from
 L<builtin>.
 
+=head2 New C<builtin::inf> and C<builtin::nan> functions
+
+The L<builtin> namespace adds two new functions, C<inf> and C<nan>, which act
+like constants that yield the floating-point infinity and Not-a-Number value
+respectively.
```
Comment (tonycoz) [inline pod/perldelta.pod]: `builtin` adds only those two new functions?

Maybe:
```
Two new functions, C<inf> and C<nan>, have been added to the C<builtin> namespace.
```

Comment (leonerd) [reply pod/perldelta.pod]: fixed

**Pull request merged**